app/com/gu/floodgate/contentsource/ContentSource.scala

```diff
@@ -13,16 +13,29 @@ case class ContentSourcesResponse(contentSources: Seq[ContentSource])
 case class SingleContentSourceResponse(contentSource: ContentSource)
 
 @JsonFormat
-case class ContentSource(id: String = UUID.randomUUID().toString,
+case class ContentSource(id: String,
+  appName: String,
+  description: String,
+  reindexEndpoint: String,
+  environment: String)
+
+@JsonFormat
+case class ContentSourceForUpdates(
```

Comment:
The difference is that when we update we don't allow them to update the environment and therefore this is not included.